### ISSUE TITLE
Ethereum Cat Herder Meeting #54

### ISSUE THREAD

**Issue by poojaranjan** [ethereum-cat-herders/PM#165]
## Date and Time

Mar 16, 2021, at 15:00 UTC

## Location 

Zoom: https://us02web.zoom.us/j/84834303064?pwd=YTVDNUxvSjEzL1VTOXNoNmI1a2RlZz09

Video at YouTube: https://www.youtube.com/channel/UCD9iiIwTRtLDYcEWONs2Q3A

## Agenda

New members introduction

### 3. Events & community meetings
- Berlin upgrade community call 

### 4. [Meeting notes](https://docs.google.com/spreadsheets/d/1axM1KaKc-GCcoPbAnYTOhgLnxXsBVoO5XRjTN3CLnck/edit#gid=0) 
- [All Core Dev meeting #107](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%20107.md)
- Eth2 Implementers' meeting #59
- [EIP Improvement Process meeting #28](https://github.com/ethereum-cat-herders/EIPIP/blob/091a0f37085653114cb043c5c0964fdb403a7fe5/All%20EIPIP%20Meetings/Meeting%20028.md)

### 5. [Peep an EIP](https://www.youtube.com/playlist?list=PL4cwHXAawZxqu0PKKyMzG_3BJV_xZTi1F) 
- [EIP-3074: Native Sponsored Transactions with Sam Wilson](https://youtu.be/4A3N8dvdaSk)
- [EIP-3085: Wallet Add Ethereum Chain RPC Method with Eric Marks](https://youtu.be/nOIl2w33sGU)
- [EIP-2315: Simple Subroutines for the EVM with Greg](https://youtu.be/c3Zo8vPwtk0)

**Upcoming:**
- [EIP-1474: Remote procedure call specification](https://github.com/ethereum-cat-herders/PM/projects/2#card-51212856)
- [ENS EIP-137: Specs. & 2544:ENS Wildcard Resolution](https://github.com/ethereum-cat-herders/PM/projects/2#card-52433784)

### 6. Discuss and close the ECH GitHub Issues/PRs
- [ethereum-cat-herders/PM](https://github.com/ethereum-cat-herders/PM/issues)
- [ethereum-cat-herders/funding](https://github.com/ethereum-cat-herders/funding/issues)

### 7. Review of outstanding action items from previous ECH meetings
- [Meeting 53](https://github.com/ethereum-cat-herders/PM/blob/02ba0605c73892d6b5702595242db07e65824c8a/All%20Ethereum%20Cat%20Herder%20Meetings/Meeting%20053.md)

#### The date for the next meeting - Mar 30, 2021

**Comment by alita-moore** [comment]
I was unfortunately sick today so I wasn't able to make the meeting. But here's a quick update on the EIP Bot:

- the bot is done (mostly) and in the phase of integrating with the current workflow. This may take a little bit because of needing to wait on others with certain repo privileges
- I have created [an issue](https://github.com/ethereum-cat-herders/EIPIP/issues/59) on the EIPIP repo for acquiring feature requests from the EIP community for bot features. I did this because there was talk about more features being wanted.
